utils/htmlToPDF.ts

```diff
@@ -0,0 +1,23 @@
+import { create, CreateOptions } from "html-pdf";
+import { readFileSync } from "fs";
+
+/**
+ * converts hard coded html into a PDF and downloads the PDF locally
+ * @param {string} downloadPath download path of the generated pdf
+ */
+const htmlToPDF = (downloadPath: string): void => {
+    // path of the hard coded html or the html as a string
+    const htmlPath = "./test.html";
```

Comment:
@brandonwong368 @feng-eric how would we do this in a real use-case? are we going to have an HTML string within our code?

Reply:
Yeah, we'll probably store a hard coded HTML string and fill it in with variables for user specific data